packages/selfcare-client-users-updater/test/selfcareClientUsersUpdaterProcessor.test.ts

```diff
@@ -0,0 +1,262 @@
+/* eslint-disable prefer-const */
+/* eslint-disable functional/no-let */
+import { randomUUID, UUID } from "node:crypto";
+import {
+  vi,
+  afterEach,
+  beforeAll,
+  beforeEach,
+  describe,
+  MockInstance,
+  it,
+  expect,
+  inject,
+} from "vitest";
+import {
+  InteropTokenGenerator,
+  RefreshableInteropToken,
+  userRole,
+} from "pagopa-interop-commons";
+import { EachMessagePayload } from "kafkajs";
+import {
+  getMockClient,
+  setupTestContainersVitest,
+} from "pagopa-interop-commons-test";
+import {
+  Client,
+  clientKind,
+  unsafeBrandId,
+  UserId,
+} from "pagopa-interop-models";
+import { selfcareClientUsersUpdaterProcessorBuilder } from "../src/services/selfcareClientUsersUpdaterProcessor.js";
+import { config } from "../src/config/config.js";
+import { AuthorizationProcessClient } from "../src/clients/authorizationProcessClient.js";
+import { relationshipStatus } from "../src/model/UsersEventPayload.js";
+import { readModelServiceBuilder } from "../src/services/readModelService.js";
+import {
+  correctEventPayload,
+  generateInternalTokenMock,
+  kafkaMessagePayload,
+} from "./utils.js";
+
+export const { cleanup, readModelRepository, postgresDB } =
+  await setupTestContainersVitest(
+    inject("readModelConfig"),
+    inject("eventStoreConfig")
+  );
+
+afterEach(cleanup);
```

Comment:
```suggestion
    undefined,
    undefined,
    undefined,
    undefined,
    undefined,
    inject("readModelSQLConfig")
  );

afterEach(cleanup);

const clientReadModelServiceSQL =
  clientReadModelServiceBuilder(readModelDB);

const oldReadModelService = readModelServiceBuilder(readModelRepository);
const readModelServiceSQL = readModelServiceBuilderSQL({
  clientReadModelServiceSQL
});
export const readModelService =
  config.featureFlagSQL &&
  config.readModelSQLDbHost &&
  config.readModelSQLDbPort
    ? readModelServiceSQL
    : oldReadModelService;
```
You need this setup in order to test the SQL too; the `eventStoreConfig` isn't needed.

I would also move all this setup in `utils`

Reply:
Which `utils` file do you mean? With what advantage?

Reply:
> Which `utils` file do you mean? With what advantage?

The one in the same folder as this, the advantage would be to have a clearer test files, in which we define only the tests, by moving all the setup in `utils`